src/scripts/settings/userSettings.js

```diff
@@ -429,6 +429,19 @@ export class UserSettings {
         return this.get('screensaver', false);
     }
 
+    /**
+     * Get or set the interval between backdrops when using the backdrop screensaver.
+     * @param {number|undefined} val - The interval between backdrops in seconds.
+     * @return {number} The interval between backdrops in seconds.
+     */
+    backdropScreensaverInterval(val) {
+        if (val !== undefined) {
+            return this.set('backdropScreensaverInterval', parseInt(val, 10), false);
```

Comment:
Do we need `parseInt`?
Since `backdropScreensaverInterval` expects a `number` (and make it consistent with the others):
```suggestion
            return this.set('backdropScreensaverInterval', val.toString(), false);
```

Reply:
I agree that the type annotation in this case is misleading at best. I think it best to leave this as is for now to be consistent with the other settings as they would need changing too. I would prefer consistency over correctness.

Reply:
It is already inconsistent. :man_shrugging: In fact, 50/50.

These two are old.
https://github.com/jellyfin/jellyfin-web/blob/ea6c8f01cfec600be45544d8ac939a8de11f85b1/src/scripts/settings/userSettings.js#L361
https://github.com/jellyfin/jellyfin-web/blob/ea6c8f01cfec600be45544d8ac939a8de11f85b1/src/scripts/settings/userSettings.js#L374

These are newer.
:point_down: _This is the first of such kind._
https://github.com/jellyfin/jellyfin-web/blob/ea6c8f01cfec600be45544d8ac939a8de11f85b1/src/scripts/settings/userSettings.js#L452
https://github.com/jellyfin/jellyfin-web/blob/ea6c8f01cfec600be45544d8ac939a8de11f85b1/src/scripts/settings/userSettings.js#L471

Reply:
Shall I just go ahead and make all of them consistent with the older style?